### DISCUSSION TITLE
Netplay not working

### DISCUSSION THREAD

**Discussion by Extreme1106** [ikemen-engine/Ikemen-GO#1331] [Q&A]
Hello, a friend and I have been trying to connect to each other via netplay for the past few days. No matter what I do, he is stuck on connecting to. I'm only able to join myself using my IPv6 address and not 127.0.0.1, he has been completely unable to join me, and every other server I host I can join easily except for my IKEMEN lobbies. What is going on?

**Comment by ZyonZZ** [comment]
> Hello, a friend and I have been trying to connect to each other via netplay for the past few days. No matter what I do, he is stuck on connecting to. I'm only able to join myself using my IPv6 address and not 127.0.0.1, he has been completely unable to join me, and every other server I host I can join easily except for my IKEMEN lobbies. What is going on?

First... I don't know what experience you have in computer networks...
but it is easy to do the test.. if you open 2 ikemens.. and you put that IP you can easily connect.. on your own pc

now....
If what you want is to connect an IP through the Internet...
You need the REAL IP that the internet gives you... and also have your PC's firewalls disabled and router... (this is as proof... then you have to give the corresponding exceptions)

I did this test and it also worked fine.

but..
ordinary users may run into problems if they lack knowledge.

**Comment by Extreme1106** [comment]
If I must clarify, I have indeed opened the port because that's Step 0, and I gave my friend my IPv4 and IPv6 address to test out. My Windows PC hosts every dedicated server that I boot up, but my Steam Deck can't join me if I host it on here. But when I try joining my Steam Deck on Windows it works just fine. I have tried hosting on both my Windows PC and my Steam Deck and my friend has not been able to connect to either. I've given them both my IPv4 address (the one that most my servers use) AND my IPv6 address (which is the only address I can use when connecting to my Steam Deck) and they have not been able to join...at all. No matter what they use.

**Comment by Extreme1106** [comment]
UPDATE: I was able to connect to my Windows PC on my Steam Deck finally but my friend still can't join
